eo-runtime/src/main/java/EOorg/EOeolang/EOcage.java

```diff
@@ -74,9 +76,22 @@ private final class Write extends PhDefault {
                 new AtComposite(
                     this,
                     rho -> {
-                        rho.attr("σ").get().attr("enclosure").put(
-                            rho.attr("x").get()
-                        );
+                        final Attr attr = rho.attr("σ").get().attr("enclosure");
+                        Phi enclosure;
+                        try {
+                            enclosure = attr.get();
+                        } catch (final EOerror.ExError error) {
+                            enclosure = null;
```

Comment:
@maxonfjvipon We are trying to avoid `null` in Java. According to: https://www.yegor256.com/2014/05/13/why-null-is-bad.html

Reply:
@maxonfjvipon Also here we got an exception swallowing. It is better to use [exception chaining](https://en.wikipedia.org/wiki/Exception_chaining) in such cases.

Reply:
@Graur I need to swallow that exception here, because I'm trying to get attribute `enclosure` which can be empty. If it's empty that means that we did not save anything into `cage` => no need to check types, just save new value